commit 52ebece1243ae6900e414b6248b5145a28348eef
Author: Guillaume Le Vaillant <glv@posteo.net>
Date:   Fri Oct 18 15:41:23 2019 +0200

    Use basic ASDF system definitions instead of package-inferred-system

diff --git a/graph.asd b/graph.asd
index 193b6e3..56afc8f 100644
--- a/graph.asd
+++ b/graph.asd
@@ -3,12 +3,10 @@
   :version "0.0.0"
   :author ("Eric Schulte <schulte.eric@gmail.com>" "Thomas Dye")
   :licence "GPL V3"
-  :class :package-inferred-system
-  :defsystem-depends-on (:asdf-package-system)
+  :in-order-to ((test-op (test-op graph-test)))
   :depends-on (alexandria
                metabang-bind
                named-readtables
                curry-compose-reader-macros
-               graph/graph))
-
-(register-system-packages "femlisp-matlisp" '(:fl.matlisp))
+               cl-heap)
+  :components ((:file "graph")))
diff --git a/graph-dot.asd b/graph-dot.asd
new file mode 100644
index 0000000..12aec7e
--- /dev/null
+++ b/graph-dot.asd
@@ -0,0 +1,8 @@
+(defsystem :graph-dot
+  :depends-on (alexandria
+               metabang-bind
+               named-readtables
+               curry-compose-reader-macros
+               cl-ppcre
+               graph)
+  :components ((:file "dot")))
diff --git a/graph-json.asd b/graph-json.asd
new file mode 100644
index 0000000..e7d091f
--- /dev/null
+++ b/graph-json.asd
@@ -0,0 +1,8 @@
+(defsystem :graph-json
+  :depends-on (alexandria
+               metabang-bind
+               named-readtables
+               curry-compose-reader-macros
+               yason
+               graph)
+  :components ((:file "json")))
diff --git a/graph-test.asd b/graph-test.asd
new file mode 100644
index 0000000..1e811e1
--- /dev/null
+++ b/graph-test.asd
@@ -0,0 +1,10 @@
+(defsystem :graph-test
+  :depends-on (alexandria
+               metabang-bind
+               named-readtables
+               curry-compose-reader-macros
+               graph
+               stefil)
+  :perform (test-op (o s)
+             (uiop:symbol-call :graph/test 'test))
+  :components ((:file "test")))
